client/src/components/Jobs/Jobs.scss

```diff
@@ -1,61 +1,73 @@
-.jobs-view {
```

Comment:
This change occurs because flex is no longer used for table layout, so the `table-layout: fixed` we had now kicks in. (See [`InfiniteScrollTable.scss`](https://github.com/TACC/Core-Portal/blob/task/FP-385-section-comp-welcome-msg--sections/client/src/components/_common/InfiniteScrollTable/InfiniteScrollTable.scss) for the relevant code.)